examples/04_06_specific_words_comparison_ctch.ipynb

```diff
@@ -0,0 +1,591 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Primerjava pristopov za luščenje ključnih besed na anotiranih besedilih iz revije Contributions to Contemporary History\n",
```

Comment:
Ni bistveno, ampak glede na to, da imamo zvezke v slovenščini: Contributions to Contemporary History so v slovenščini Prispevki za novejšo zgodovino. :)

Reply:
popravljeno